db-engine/src/main/java/com/splicemachine/db/iapi/types/DataValueFactoryImpl.java

```diff
@@ -1197,6 +1197,36 @@ public static DataValueDescriptor getNullDVDWithUCS_BASICcollation(
         }
     }
 
+        public static DataValueDescriptor getNullDVD(int typeFormatId) {
+                switch (typeFormatId) {
```

Comment:
Formatting inconsistency? How many spaces does the project use? Are there any rules to enforce them?

Reply:
@jaceklaskowski we use 4 spaces for new files, however Derby files that we merged into our tree used tabs. Over time these files ended up with a mix of tabs and spaces, which is less than ideal, but nowadays doing a repository-wide reformatting would be too invasive.